test/performance/latency-controller/README.md

```diff
@@ -1,242 +1,156 @@
-# kubestellar-latency-collector
+# Framework to Benchmark KubeStellar Data-Plane Latencies
 
-> A Kubernetes controller to measure and expose latency metrics for KubeStellar’s downsync and upsync operations.
+<p align="center">
+  <img src="images/Diagram.png" width="700" height="300" title="Benchmarking of KubeStellar  Data-Plane ">
+</p>
 
----
+## Prerequisites
 
-## 🚀 Demo Setup
+Before deploying the Latency Controller, ensure you have:
 
-Create a KubeStellar demo environment using Kind:
+1. **KubeStellar Environment**: You must have an environment with KubeStellar installed; see [KubeStellar getting started](https://docs.kubestellar.io/release-0.23.1/direct/get-started/). Alternatively, you can also use KubeStellar e2e script [run-test.sh](https://github.com/kubestellar/kubestellar/blob/main/test/e2e/run-test.sh) to setup an environment.
 
-```bash
-bash <(curl -s https://raw.githubusercontent.com/kubestellar/kubestellar/refs/tags/v0.27.2/scripts/create-kubestellar-demo-env.sh) --platform kind
-```
+2. **KubeStellar Monitoring Setup**: The base monitoring infrastructure must be installed first:
+   - Setup the KS monitoring tool [Here](https://github.com/kubestellar/kubestellar/tree/main/monitoring#readme)
+   - This installs Prometheus and Grafana components
```

Comment:
It will be better to use the following text below instead:

2. **KubeStellar Monitoring Setup**:  setup the KubeStellar monitoring stack using the instructions [Here](https://github.com/kubestellar/kubestellar/tree/main/monitoring#readme)